Fix wrong default values for cron_consumers_runner (#5566)
magento/devdocs: guides/v2.2/config-guide/mq/manage-message-queues.md

#### Behavior by default

> * Cron job `consumers_runner` is disabled

Reviewer: This is incorrect, the default is `enabled`, please revert.

> * Cron job `consumers_runner` is disabled

Reviewer 2: ```suggestion
* Cron job `consumers_runner` is enabled
```

* Cron job `consumers_runner` runs all defined consumers

> * Each consumer processes 1000 messages and then terminates

Reviewer: You may update this line though, the default number of messages is 10000, not 1000.

...
```

> * `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is false.

Reviewer 2: ```suggestion
*  `cron_run` - A boolean value that enables or disables the consumers_runner cron job (default = `false`).
```

> * `max_messages` - the maximum number of messages for each consumer that must be processed before consumer terminate, by default is 1000. If it is 0, then the consumer never stops working.

Reviewer 2: ```suggestion
* `max_messages` - A number specifying the maximum number of messages each consumer must process before terminating (default = `1000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.
```

Reviewer 3: This comment applies to both versions of this file.

@meker12 I disagree with this suggestion "A number specifying the maximum number"

Also, a maintainer or architect needs to approve the recommendation to not use 0 as a value. That was a feature when message queues were introduced, but now RabbitMQ is the preferred solution for on-premise installations.

Author: Hi @keharper,

 The same info was recommended in [Deploy variables](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#cron_consumers_runner) for Magento Cloud. I think @meker12 is just maintaining the compatibility of the informations.

Reviewer 2: > This comment applies to both versions of this file.
> 
> @meker12 I disagree with this suggestion "A number specifying the maximum number"
> 
> Also, a maintainer or architect needs to approve the recommendation to not use 0 as a value. That was a feature when message queues were introduced, but now RabbitMQ is the preferred solution for on-premise installations.

What do you disagree with in reg

> This comment applies to both versions of this file.
> 
> @meker12 I disagree with this suggestion "A number specifying the maximum number"
> 

@keharper What do you suggest as an alternative?

> Also, a maintainer or architect needs to approve the recommendation to not use 0 as a value. That was a feature when message queues were introduced, but now RabbitMQ is the preferred solution for on-premise installations.

I was matching the Cloud description for this variable, but perhaps this is incorrect. Who is the correct maintainer to review?

There were a number of changes to the CONSUMERS_RUNNER processing released with the Cloud ece-tools package update.  Some of the associated documentation changes to Core were introduced in this PR:  https://github.com/magento/devdocs/pull/5289/, which was merged into the `develop` branch.

Reviewer 3: The simplest option: delete "A number specifying"

> * `consumers` - the list of consumers which will be run, by default is empty array (all consumers are allowed to be run).

Reviewer 2: ```suggestion
* `consumers` - An array of strings specifying which consumer(s) to run. An empty array runs all consumers.
```

...
```

> * `cron_run` - A boolean value that enables or disables the consumers_runner cron job (default = `false`).

Reviewer 2: ```suggestion
* `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `false`).
```

> * `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `false`).

Reviewer 2: ```suggestion
* `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `true`).
```

> * `max_messages` - A number specifying the maximum number of messages each consumer must process before terminating (default = `1000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.

Reviewer 2: ```suggestion
* `max_messages` - The maximum number of messages each consumer must process before terminating (default = `10000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.
```
